framework/src/media/Kconfig

```diff
@@ -78,24 +78,20 @@ config AUDIO_RESAMPLER_BUFSIZE
 	---help---
 		Buffer size for resampler
 
-config FILE_DATASOURCE_STREAM_BUFFER_SIZE
-	int "File DataSource stream buffer size"
+config CONFIG_HANDLER_STREAM_BUFFER_SIZE
```

Comment:
1. This causes `CONFIG_CONFIG_XXX` but you use `CONFIG_XXX` in source code. I am sure you don't want this.
2. From config name, I can't image where it belongs. How about add MEDIA prefix at name?